test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -674,14 +675,7 @@ public static boolean assertionsEnabled() {
     /**
      * Asserts that there are no files in the specified path
      */
-    public void assertPathHasBeenCleared(String path) throws Exception {
-        assertPathHasBeenCleared(PathUtils.get(path));
-    }
-
-    /**
-     * Asserts that there are no files in the specified path
-     */
-    public void assertPathHasBeenCleared(Path path) throws Exception {
+    public void assertPathHasBeenCleared(Path path) {
```

Comment:
At the moment it's only used in these (few) integration tests. I was debating this as well, but I think the method is also useful in unit tests.